Draggable/explainer.md

```diff
@@ -0,0 +1,165 @@
+            animation-iteration-count: infinite;
+            app-region: drag; /* <-------- APP-REGION */
+        }
+
+        @keyframes moveDown {
+            from { margin-top: 0px; }
+            to { margin-top: 600px;}
+        }
+    </style>
+</head>
+<body>
+    <button title="should be draggable">
+        Generic Beautiful Button
+    </button>
+    
+</body>
+</html>
+```
+
+In this example, the 'WCO' feature is enabled, hence the window has lost the meaningful area to be moved across the screen.
+
+The (Generic Beautiful) button has the `app-region` set to `drag`. This button can how be used to drag the window around.
+
+> Note: This is just an example to test the _current_ behaviour on elements that have animations. the `app-region` is generally used on emulated titlebars or other UX designed to let the end user move the window around.
+
+### `app-region` and pointer events
+
+Current implementations of `app-region` **ignore pointer events**. Interactive elements inside an area defined with `app-region: drag` will require a `app-region: no-drag` to be interacted with.
+
+### Suppression of `app-region`
+
+The `app-region` property applies to a DOM element, and allows it to become a point from where to drag and move a window. If this element's `visibility` is `hidden` or `collapsed`, or if the element is occluded by an element with a higher `z-index` then the window can't be dragged from that element.
+
+### `app-region` and animations
+The property works and while the user continues to keep their pointer down, even of the animation has moved the element away from where that pointer is, they can continue with the window movement they started. 
+
+## Sample use case
+
+A sample use case is a web application that is using [Window Controls Overlay](https://developer.mozilla.org/en-US/docs/Web/API/Window_Controls_Overlay_API) to create a custom titlebar. Elements like buttons and textboxes that need to be interacted with use the `no-drag` value to be accessed. Everything else in the `#titlebar` style will be a point from where the user can drag and move the window.
+
+```css
+/* Custom titlebar style */
+#titlebar {
+    width: env(titlebar-area-width, 100%);
+    height: env(titlebar-area-height, 32px);
+    top: env(titlebar-area-y, 0px);
+    left: env(titlebar-area-x, 0px);
+    ...
+    -webkit-app-region: drag;
+    app-region: drag;
+}
+
+/*A search textbox inside the custom titlebar*/
+#searchBox {
+  app-region: no-drag; /* allows the textbox to be interacted with*/
+}
+```
+## Stakeholder Feedback / Opposition
+The process to standardize this feature was logged in an [issue #7017](https://github.com/w3c/csswg-drafts/issues/7017) in 2022. This explainer aims at restarting the conversation. 
+
+Note that there is [positive](https://github.com/w3c/csswg-drafts/issues/7017#issuecomment-1609114145) developer signals in the issue.
+
+## Accessibility, Privacy, and Security Considerations
+There are no considerations at this time.
```

Comment:
It would be good to call out how this does (or doesn't) impact accessibility. My initial thoughts would be that it doesn't block any existing operations, but I'm curious if work needs to be specified to make a custom title bar area present the same as a traditional title bar to an accessible user?

For security / open questions - how should this be restricted? Can it be in a shadow DOM? iframe? middle of the page?
  Continuing from that for Privacy - if the locations for a valid app-region are restricted by the browser, will the site be able to distinguish between a functioning/proper app-region and an invalid one? And if so, what kind of fingerprinting could this expose?

Reply:
I will expand on these questions. Need to check what happens when this is present in an iframe, for example..